library/asn1parse.c

```diff
@@ -180,6 +181,20 @@ int mbedtls_asn1_get_int( unsigned char **p,
     return( 0 );
 }
 
+int mbedtls_asn1_get_int( unsigned char **p,
+                    const unsigned char *end,
+                    int *val )
+{
+    return asn1_get_tagged_int (p, end, MBEDTLS_ASN1_INTEGER, val);
```

Comment:
Style:
```suggestion
    return( asn1_get_tagged_int (p, end, MBEDTLS_ASN1_INTEGER, val) );
```
and likewise for `get_enum` and for the write functions

Reply:
Done.